### PR TITLE
Switch perf playground to measure void align branch

### PR DESCRIPTION
**Pull request by bradcray**

### CODE REVIEW

**Comment by bradcray** [comment]
@ronawho: Would you spot-check that I've updated this properly to measure #7205?

File: util/cron/test-perf.chapcs.playground.bash [hunk 1]
```diff
@@ -10,10 +10,10 @@ source $CWD/common-perf.bash
 export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.chapcs.playground"
 
 # Test performance of implementation changes in forall intents
```
Comment (ronawho) [inline util/cron/test-perf.chapcs.playground.bash]: Comment is out of date now (though obviously it won't actually impact testing)

**Comment by bradcray** [comment]
@ronawho: I updated and added instructions in the form of comments to help future editors.

**Comment by ronawho** [comment]
These instructions only work for the really simple case of testing a branch with no merge-conflicts, but since that's the most common case we test, I'm fine with leaving them in

**Comment by bradcray** [comment]
I could add a caveat to this effect and add a "call Elliot" note for cases that do have merge conflicts.  :)

**Comment by ronawho** [comment]
>I could add a caveat to this effect and add a "call Elliot" note for cases that do have merge conflicts. :)

:)

**Pull request merged**